components/cloud-foundry/frontend/src/model/route.model.js

```diff
@@ -25,6 +25,8 @@
    */
   function Route(apiManager, modelUtils) {
 
+    var routesApi = apiManager.retrieve('cloud-foundry.api.Routes');
```

Comment:
We've had some issues in the past where the apiManager/modelManager cannot retrieve the objects when their host function is run first, so will grab them when needed. It's probably best to leave these as is.